### DISCUSSION TITLE
프로세스 상태 다이어그램을 그려주세요!

### DISCUSSION THREAD

**Discussion by Sleepingoff** [OurStudyLog/operatingsystem#17] [Q&A] [answered]
프로세스 상태
- 생성
- 준비
- 대기
- 실행
- 종료

**Comment by atoonatoo** [comment]
![image](https://github.com/OurStudyLog/operatingsystem/assets/129242580/9d6a171c-5a18-40d4-a89a-8cde891231c4)

**Comment by ImSeon01** [comment, marked as answer]
![image](https://github.com/OurStudyLog/operatingsystem/assets/121650379/2bb500f5-d236-4553-bb0f-7294bfcbbf42)

**Comment by Sleepingoff** [comment]
![image](https://github.com/OurStudyLog/operatingsystem/assets/111721483/266b58fd-a6d3-4c0f-a609-07a60b4194e2)
